x/ibc/03-connection/keeper/keeper.go

```diff
@@ -96,7 +95,7 @@ func (k Keeper) SetClientConnectionPaths(ctx sdk.Context, clientID string, paths
 func (k Keeper) addConnectionToClient(ctx sdk.Context, clientID, connectionID string) sdk.Error {
 	conns, found := k.GetClientConnectionPaths(ctx, clientID)
 	if !found {
-		return types.ErrClientConnectionPathsNotFound(k.codespace, clientID)
+		conns = []string{}
```

Comment:
?

Reply:
It might not have been initialised I think, cc @mossid